pkg/policy/consumer.go

```diff
@@ -211,8 +206,8 @@ func (c *Consumable) removeFromMaps(id NumericIdentity) {
 // Returns true if the identity was not present in this Consumable's
 // IngressIdentities map, and thus had to be added, false if it is already added.
 func (c *Consumable) AllowIngressIdentityLocked(cache *ConsumableCache, id NumericIdentity) bool {
-	isIdentityAllowed := c.isIdentityAllowed(id)
-	if isIdentityAllowed == false {
```

Comment:
What happens if the identity exists but is not allowed? Wait, why was the function `isIdentityAllowed` removed? I think it's better to leave it, it's more simple to call the function without worrying about what's the meaning of the value/keys of a map

Reply:
> Defer adds overhead in small functions. Is is ok if it goes like this?

I have no strong opinion on this, so I'll revert that change.

Reply:
> What happens if the identity exists but is not allowed? Wait, why was the function isIdentityAllowed removed? I think it's better to leave it, it's more simple to call the function without worrying about what's the meaning of the value/keys of a map

I disagree. Now it's just a check for existence in the map, which is provided by Golang. I don't see why we need to add a wrapper around something that Golang provides out of the box with just one line.

Reply:
I had to read th whole code. Makes sense